src/Filters/WhereNull.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/*
+ * Copyright 2022 Cloud Creativity Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace LaravelJsonApi\Eloquent\Filters;
+
+use LaravelJsonApi\Core\Support\Str;
+use LaravelJsonApi\Eloquent\Contracts\Filter;
+
+class WhereNull implements Filter
+{
+    use Concerns\HasColumn;
+    use Concerns\IsSingular;
+
+    /**
+     * @var bool
+     */
+    private bool $not;
+
+    /**
+     * Create a new filter.
+     *
+     * @param string $key
+     * @param bool $not
+     * @param string|null $column
+     * @return static
+     */
+    public static function make(string $key, bool $not, string $column = null): self
```

Comment:
Remove the `$not` from this method, as it doesn't logically match up with other filter `make` methods. Instead, have the not property have a default value (either true or false), then add a method that the developer can call to change the value of `not`.